package.json

```diff
@@ -54,7 +56,9 @@
     "reactstrap": "5.0.0-alpha.4",
     "redux": "3.7.2",
     "redux-debounce": "1.0.1",
-    "taskcluster-client": "2.5.4"
+    "superagent-promise": "^1.1.0",
```

Comment:
`superagent-promise` doesn't appear to be directly by Treeherder, so unless it's a peerDependency of something can be removed.

Reply:
It's deprecated now, anyway -- superagent uses promises directly.